disnake/ui/button.py

```diff
@@ -88,6 +88,17 @@ class Button(Item[V]):
     # We have to set this to MISSING in order to overwrite the abstract property from WrappedComponent
     _underlying: ButtonComponent = MISSING
 
+    @overload
+    def __new__(cls: Type[Button[None]], **kwargs) -> Button[None]:
+        ...
+
+    @overload
+    def __new__(cls: Type[Button[V]], **kwargs) -> Button[V]:
+        ...
+
+    def __new__(cls: Type[Button[Any]], **kwargs) -> Any:
+        return super().__new__(cls)
```

Comment:
The explicit None is what makes `Button()` default to type `Button[None]`. Without this, it would remain `Button[Unknown]`. I suppose the ideal solution would be to have buttons not be inherently dependent on views (i.e. have a `Button` base class and use `Item` as a mixin for, say, a `ViewButton[V]` class), but that would mean we get yet another layer of classes for each component.